com.microsoft.azuredatastudio.yml

```diff
@@ -0,0 +1,47 @@
+app-id: com.Microsoft.AzureDataStudio
+tags:
+  - proprietary
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: azuredatastudio.sh
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --share=network
+  - --filesystem=host
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+modules:
+  - name: azuredatastudio
+    buildsystem: simple
+    build-commands:
+      - install -D azuredatastudio.sh /app/bin/azuredatastudio.sh
+      - install -D azuredatastudio /app/bin/azuredatastudio
+      - install -D com.Microsoft.AzureDataStudio.appdata.xml /app/share/metainfo/com.Microsoft.AzureDataStudio.metainfo.xml
+      - install -Dm755 chrome_100_percent.pak /app/bin/chrome_100_percent.pak
+      - install -Dm755 chrome_200_percent.pak /app/bin/bin/chrome_200_percent.pak
+      - install -Dm755 chrome_crashpad_handler /app/bin/chrome_crashpad_handler
+      - install -Dm755 icudtl.dat /app/bin/icudtl.dat
+      - install -Dm755 libEGL.so /app/bin/libEGL.so
+      - install -Dm755 libffmpeg.so /app/bin/libffmpeg.so
+      - install -Dm755 libGLESv2.so /app/bin/libGLESv2.so
+      - install -Dm755 libvk_swiftshader.so /app/bin/libvk_swiftshader.so
+      - install -Dm755 libvulkan.so.1 /app/bin/libvulkan.so.1
+      - install -Dm755 resources.pak /app/bin/resources.pak
+      - install -Dm755 snapshot_blob.bin /app/bin/snapshot_blob.bin
+      - install -Dm755 v8_context_snapshot.bin /app/bin/v8_context_snapshot.bin
+      - install -Dm755 vk_swiftshader_icd.json /app/bin/vk_swiftshader_icd.json
+      - cp -rf locales/ /app/bin/locales
+      - cp -rf resources /app/bin/resources
+      
+      - install -Dm644 com.Microsoft.AzureDataStudio.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 ${FLATPAK_ID}.png /app/share/icons/${FLATPAK_ID}.png
+      - desktop-file-edit --set-icon=${FLATPAK_ID} /app/share/applications/${FLATPAK_ID}.desktop
+      - appstream-util validate-relax /app/share/metainfo/com.Microsoft.AzureDataStudio.metainfo.xml
+    sources:
+      - type: dir
```

Comment:
this will not work. `dir` sources are to be avoided. but then there is nothing here that match.

There should be a URL to get pre-build binaries.